addons/payment_mollie_official/models/payment_icon.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, api
+
+from mollie.api.client import Client
+from odoo.addons.payment_mollie_official.models.payment_acquirer_method import get_mollie_provider_key
+
+# minimum and maximum amounts per payment method
+DEFAULT_METHOD_VALUES = {
```

Comment:
Why is the extension of `payment.icon` model necessary if you have this? Should this be sufficient?